sui/its.js

```diff
@@ -124,8 +235,37 @@ if (require.main === module) {
             mainProcessor(removeTrustedAddress, options, [trustedChain], processCommand);
         });
 
+    const allowFunctionsProgram = new Command()
+        .name('allow-functions')
+        .description('Allow functions')
+        .command('allow-functions <versions> <functions>')
+        .action((versions, functions, options) => {
+            mainProcessor(allowFunctions, options, [versions, functions], processCommand);
+        });
+
```

Comment:
create a separate `contract.js` script that allows generically allowing/disallowing function by version for any of our contract. It can do also support `pause --all`, `unpause --all`. And for the curated `pause` and `unpause` we can create an object mapping for contract name -> curated function list